cypress/support/built-script.ts

```diff
@@ -0,0 +1,9 @@
+import * as PurpleDot from '../../src/index';
```

Comment:
To make this more e2e, is there a way to actually install the package as a dependency and use it like an integration would?

Reply:
Cypress has a feature called "[Component Testing](https://docs.cypress.io/guides/component-testing/overview)" which might be more appropriate for this use case.

Reply:
For installing the actual package:
- I did start with importing, but you need to get the code into the window as opposed to just the cypress window surrounding the window
- To install the package we would need the package to see it as another package, which I might be able to do? The build step is the main thing though and this gets pretty close

- I did try the component testing but it looked like you needed to pick React or Vue and then it gives you something to mount into to test individual React or Vue components so didn't seem like a good fit